gradle/dependencies.gradle

```diff
@@ -42,7 +42,7 @@ ext {
       commonsIO                 : '2.6',
       robolectric               : '4.3.1',
       mockwebserver             : '3.12.0',
-      lifecycle                 : '2.0.0-rc01',
+      lifecycle                 : '2.2.0',
```

Comment:
I decided to update the version of lifecycle to 2.2.0. Didn't have any specific reason, just saw that before it was old RC version, while now 2.2.0 stable is available